src/components/Recipes/RecipeItem.tsx

```diff
@@ -38,11 +39,32 @@ const RecipeItem: React.FC<{recipeID: number, text: string, image: string, }> =
     });
   };
 
+  const searchResult = searchResultInfo.filter((recipeItem) => props.recipeID === recipeItem.id)[0]
+
+  const missingIngredientList = searchResult.missedIngredients.map((missedIngredient) => {
+    return <li key={missedIngredient.id}>{missedIngredient.amount} {missedIngredient.unitLong} {missedIngredient.name}</li>
+  });
+
+  const usedIngredientList = searchResult.usedIngredients.map((usedIngredient) => {
+    return <li key={usedIngredient.id}>{usedIngredient.amount} {usedIngredient.unitLong} {usedIngredient.name}</li>
+  });
+
   return (
     <li key={props.recipeID}>
       <Card class='recipeCard'>
         <img className="card-header" src={props.image} />
         <h2>{props.text}</h2>
+
+        <div className="missingIngredients">
```

Comment:
in html, naming conventions are usually snake-case instead of camelCase like in react

also if this is a style container, making it apparent it is a container is a good choice ('ingredient-container')